integration-tests/check-subscriptions

```diff
@@ -109,6 +111,22 @@ def machine_python(machine, script, arg=""):
     return machine.execute("python3 -c %s %s" % (shell_escape(script), shell_escape(arg)))
 
 
+def skipUnlessDistroFamily(distro, reason):
```

Comment:
FWIW, I'm happy to add something like that to the official `skipImage()`, maybe make it accept `re.Pattern` objects in addition to strings?

Reply:
Though about that, didn't come up with something that could make sense for cockpit in general. Will think about it more.